56.md

```diff
@@ -2,35 +2,38 @@
 NIP-56
 ======
 
-Reporting
+Labeling
 ---------
 
-`draft` `optional` `author:jb55`
+`draft` `optional` `author:jb55` `author:michaelhall923` `author:gruruya`
 
-A report is a `kind 1984` note that is used to report other notes for spam,
-illegal and explicit content.
+A label is a `kind 1984` note that describes the content of other notes or pubkeys.  
+It's a bit similar to a `t` tag, except it can be added by others.
 
 The content MAY contain additional information submitted by the entity
-reporting the content.
+labeling the content.
 
 Tags
 ----
 
-The report event MUST include a `p` tag referencing the pubkey of the user you
-are reporting.
+The label event contains one or more tags:
+- `[e, <event-id>, <label>]` - Label event
+- `[p, <pubkey>]` - Use pubkey to find recommended relays
+- `[p, <pubkey>, <label>]` - Same as above, but also label pubkey
 
-If reporting a note, an `e` tag MUST also be included referencing the note id.
+The `<label>` may optionally be followed by a `<classification>` such as:
+- `"content"` - The content is `label`
+- `"image"` - The image depicts `label`
+- `"topic"` - The event/profile is about `label`
 
-A `report type` string MUST be included as the 3rd entry to the `e` or `p` tag
-being reported, which consists of the following report types:
+Following the classification, there may be an `<affirmative marker>` which must be one of the following:
+- `"+"` - The `label` is accurate
+- `"-"` - The `label` is inaccurate
```

Comment:
What is the reasoning for this "affirmative marker"? I imagine that accuracy of labels can be determined by how many users labeled something with the same label.

Reply:
Many services use negative voting, I want to avoid multiple different ad-hoc solutions such as prefixing the label with a `!` or a `-`. It doesn't have to be used, I'll add a section that says a client may choose to ignore negative affirmative markers.

Reply:
Ahh, I see. I thought by mistake that "accurate" means here that the user is sure this is the correct label. And "inaccurate" means that the user is not sure.

But "accurate" means "yes, this is the correct label" vs. "no, this is the wrong label" for "inaccurate"

Not a native English speaker haha

Reply:
I wasn't sure how to label this either, so that's helpful feedback.

What about "a vote for" or "believes to be accurate"

Reply:
The new description is great!